README.md

```diff
@@ -1,2 +1,84 @@
 # cephmetrics
-ceph metric collectors with collectd integration
+
+Cephmetrics is a tool that allows a user to visually monitor various metrics in a running Ceph cluster.
+
+## Disclaimer
+
+This document contains the installation procedure for cephmetrics released on June 30, 2017.
+
+## Prerequisites
+- RHEL 7 should be running on all hosts
+- A functional ceph cluster running version ceph-osd-10.2.7-27.el7cp.x86_64 or later is already up and running.
+- Another host machine independent of the ceph machines must be available.  This host will be used to receive data pushed by the hosts in the Ceph cluster, and will run the dashboard to display that data.
+- A host machine on which to execute `ansible-playbook` to orchestrate the deployment must be available.  In this document, this will be the same host as the dashboard host.
+- Passwordless SSH access from the deploy host to the ceph hosts. The username should be the same for all hosts.
+- Passwordless sudo access on the ceph and dashboard hosts
+- All hosts must share the same DNS domain
+
+## Expected configuration
+
+After running this procedure, the following configuration is expected.
+- The ceph nodes will basically remain unchanged, except that repos necessary for interacting with the cephmetrics host machine will be installed.
```

Comment:
"The ceph nodes will have `collectd` installed, along with collector plugins from `cephmetrics-collectd`"